app/src/main/java/org/eyeseetea/malariacare/fragments/AVFragment.java

```diff
@@ -152,4 +156,17 @@ public void onClick(Media media) {
     public void reloadData() {
         initializePresenter();
     }
+
+    public void showProgress() {
+        mainExecutor.run(new Runnable() {
+            @Override
+            public void run() {
+                if(PreferencesState.getInstance().isFileDownloaderInProgress()) {
```

Comment:
AvFragment should not invoke PreferenceState because is in data layer, better if according to current code we create two methods showDownloadProgress and hideDownloadProgress to invoke from DashboardActivityStrategy